src/inc/corjit.h

```diff
@@ -161,6 +161,31 @@ class CORJIT_FLAGS
 
         CORJIT_FLAG_NO_INLINING             = 42, // JIT should not inline any called method into this method
 
+#if defined(_TARGET_ARM64_)
+
+        CORJIT_FLAG_HAS_AES                 = 43, // ID_AA64ISAR0_EL1.AES is 1 or better
```

Comment:
Is it worthwhile either:
* Making this have the same name as the equivalent X86 flag (`CORJIT_FLAG_USE_AES`) -- or vice versa
-or-
* Giving this a distinct name that clearly identifies it as an ARM flag (and do the same for the X86 flags)

?